packages/react/src/components/Tree/TreeItem.tsx

```diff
@@ -65,9 +70,14 @@ export interface TreeItemProps extends UIComponentProps, ChildrenComponentProps
 class TreeItem extends UIComponent<ReactProps<TreeItemProps>> {
   static create: Function
 
+  static displayName = 'TreeItem'
+
   static className = 'ui-tree__item'
 
-  static displayName = 'TreeItem'
+  static slotClassNames: TreeItemSlotClassNames = {
+    title: TreeTitle.className,
+    tree: `${TreeItem.className}__tree`,
```

Comment:
seems that this part rather have a `subtree` semantics. The reason I am worrying about that is because client might wrongly think that this selector is for tree itself, not for its subtree part. May we rename it?

Reply:
let's see also what's @mnajdova 's take on this, we were discussing it for `submenu` vs `menu` so we need to align

Reply:
discussed with @mnajdova as well and applied @kuzhelov 's suggestion